numpy/fft/_helper.py

```diff
@@ -121,14 +122,13 @@ def ifftshift(x, axes=None):
 
     return roll(x, shift, axes)
 
-
 @set_module('numpy.fft')
 def fftfreq(n, d=1.0, device=None):
     """
     Return the Discrete Fourier Transform sample frequencies.
 
     The returned float array `f` contains the frequency bin centers in cycles
-    per unit of the sample spacing (with zero at the start).  For instance, if
+    per unit of the sample spacing (with zero at the start). For instance, if
```

Comment:
In general, please don't fix this type of thing.  This is perfectly correct, just different habit/style.